halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/azure/AzureAddAccountCommand.java

```diff
@@ -77,17 +77,31 @@ protected String getProviderName() {
   )
   private String defaultKeyVault;
 
+  @Parameter(
+      names = "--packer-resource-group",
+      description = AzureCommandProperties.PACKER_RESOURCE_GROUP_DESCRIPTION
+  )
+  private String packerResourceGroup;
+
+  @Parameter(
+      names = "--packer-storage-account",
+      description = AzureCommandProperties.PACKER_STORAGE_ACCOUNT_DESCRIPTION
+  )
+  private String packerStorageAccount;
+
   @Override
   protected Account buildAccount(String accountName) {
     AzureAccount account = (AzureAccount) new AzureAccount().setName(accountName);
 
```

Comment:
It's a lombok issue, `setName()` is defined on `Account`, and the other setters are defined on `AzureAccount`.